features/doctrine/search_filter.feature

```diff
@@ -717,6 +717,45 @@ Feature: Search filter on collections
     }
     """
 
+  @createSchema
+  Scenario: Get collection by ulid 01H2ZS93NBKJW5W4Y01S8TZ43M
+    Given there is a UidBasedId resource with id "01H2ZS93NBKJW5W4Y01S8TZ43M"
+    When I send a "GET" request to "/uid-based-ids?id=/uid-based-ids/01H2ZS93NBKJW5W4Y01S8TZ43M"
```

Comment:
```suggestion
    When I send a "GET" request to "/uid_based_ids?id=/uid-based-ids/01H2ZS93NBKJW5W4Y01S8TZ43M"
```

Reply:
Thanks for spotting this! I've been using kebab-case for years, I didn't remember it was not the standard 🙃